test/Notifier-test.ts

```diff
@@ -26,6 +26,10 @@ import {
     SyncState,
 } from "matrix-js-sdk/src/matrix";
 import { waitFor } from "@testing-library/react";
+// eslint-disable-next-line no-restricted-imports
+import { MatrixRTCSession } from "matrix-js-sdk/src/matrixrtc/MatrixRTCSession";
+// eslint-disable-next-line no-restricted-imports
+import { CallMembership } from "matrix-js-sdk/src/matrixrtc/CallMembership";
```

Comment:
Ditto. Seems like `matrixrtc` should have an index with all its public exports and we should allowlist that and consume that

Reply:
Thanks for making me finally take care of this...
https://github.com/matrix-org/matrix-js-sdk/pull/4314
and
https://github.com/matrix-org/matrix-react-sdk/pull/12780

I dont think this fits into this PR and I would like to not block this PR on the js-sdk fix (its rather annoying to not have the ringing stop)

So my proposal would be to merge this without the lint rules and I keep the other PR's ontop of this so we can review and merge them independently?